recipes/mikeio/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "mikeio" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0eda011a42863e998df55ae7b9e324b3c470f2d5ce6f56feb2d2ae082492eff8
+
+build:
+  skip: true  # [not win or py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pythonnet
+    - numpy
+    - pandas
+    - scipy
+    - pyyaml
+    - shapely
+    - xarray
+    - matplotlib-base
+
+test:
+  imports:
+    - mikeio
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest -v tests
+
+about:
+  home: https://github.com/DHI/mikeio
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: License.txt
```

Comment:
It looks like the license file has been recently been added to the upstream `MANIFEST.in`, but it is not distributed in the current release. Please manually package the license file for now, and then when the next releases comes out you can remove it when you update the feedstock.

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Reply:
@synapticarbors thank you, this is a good idea. Now we need to resolve tests.

Reply:
It looks like the test data is not included in the source distribution. This would need to be fixed upstream. That said, conda-forge, doesn't require that you run the tests. The imports showing existence are usually enough. If you want to remove running the test suite for now, I can merge and then once the files are in-place in a future release, the feedstock can be updated to then run the tests again.